Formula/clasp-cl.rb

```diff
@@ -0,0 +1,34 @@
+class ClaspCl < Formula
+  desc "Common Lisp implementation that brings Common Lisp and C++ together"
+  homepage "https://github.com/clasp-developers/clasp"
+  url "https://github.com/clasp-developers/clasp/archive/refs/tags/2.0.0.tar.gz"
+  sha256 "de3c5efaad0ca1e87ce58e937a45f2740d0c5cd869a500b606eb9e2ea37cb837"
+  license "GPL-2.0-or-later"
+  head "https://github.com/clasp-developers/clasp.git",
+       using:  :git,
+       branch: "main"
+
+  depends_on "boost" => :build
+  depends_on "libunwind-headers" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "sbcl" => :build
+  depends_on arch: :x86_64
+  depends_on "fmt"
+  depends_on "gmp"
+  depends_on "llvm@14"
+
+  def install
+    ENV.deparallelize
+    system "./koga", "--reproducible-build", "--bin-path=#{bin}", "--share-path=#{share}/clasp/",
+      "--lib-path=#{lib}/clasp/", "--llvm-config=#{Formula["llvm@14"].opt_bin}/llvm-config",
+      "--pkg-config=#{Formula["pkg-config"].opt_bin}/pkg-config",
+      "--cxxflags=-I#{Formula["boost"].include}/", "--cppflags=-I#{Formula["boost"].include}/",
+      "--skip-sync=ansi-test,mps,cl-bench,cl-who"
+    system "ninja", "-C", "build", "install"
+  end
+
+  test do
+    assert_match "clasp-boehmprecise", shell_output("#{bin}/clasp --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Ok. I'll fix that.

Reply:
This should be fixed